SerratusApi/Model/PaginatedResult.cs

```diff
@@ -0,0 +1,15 @@
+﻿using SerratusTest.Domain.Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SerratusApi.Model
+{
+    public class PaginatedResult
+    {
+        public IEnumerable<AccessionSection> AccessionSections { get; set; }
```

Comment:
having both sections in here can be confusing - they are used in a mutually exclusive way. how about a generic typed prop?

```cs
public class PaginatedResult<T>
{
    public IEnumerable<T> Items { get; set; }
    public int NumberOfPages { get; set; }
}
```

This would be used like:
```cs
var paginatedResult = new PaginatedResult<AccessionSection>
{
    Items = accs,
    NumberOfPages = numPages
};
```